tracer/build/_build/Build.Steps.Debugger.cs

```diff
@@ -77,6 +81,11 @@ partial class Build
         {
             DotnetBuild(DebuggerSamples, framework: Framework);
 
+            if (ExceptionReplaySamples.TryGetTargetFrameworks().Contains(Framework))
```

Comment:
Why do we need to compile it separately from other samples?

Reply:
No need to compile it separately, but the sample of Exception Replay is targeting _most_ of the runtimes, except .NET Core 2.1 that is marked EOL and dropped support for in the upcoming major version of `dd-trace-dotnet` V3.0.

Reply:
I see. Then, this build  code should be in `CompileDebuggerIntegrationTestsDependencies`? 🤔

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/5821#discussion_r1721645653